src/api/app/views/webui2/webui/project/show.html.haml

```diff
@@ -24,10 +24,24 @@
     = render partial: 'webui2/shared/buildresult_box', locals: { project: @project.name }
   .list
     .card
-      %h5.card-header
-        Packages
-        = "(#{@packages.length})"
-      = render partial: 'project_packages'
+      .bg-light
+        %ul.nav.nav-tabs.pt-2.px-3.flex-nowrap{ role: 'tablist', id: 'packages-tabs' }
+          %li.nav-item
+            %a.nav-link.active{ id: 'packages-tab', href: '#packages', role: 'tab', data: { toggle: 'tab' },
+            aria: { controls: 'packages', selected: 'true' } }
+              Packages
+              = "(#{@packages.length})"
```

Comment:
You can write line 32 and 33 as:

`Packages (#{@packages.length})`

I would expect the linters to complain currently in line 33